ansible/files/ocp/namespace/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: openstack
```

Comment:
Installing our operators via OLM will create the namespace for us (via the OperatorGroup). I'm not sure we'll need this namespace.yaml file.